bootstrap/bootstrap.ps1

```diff
@@ -0,0 +1,107 @@
+param
+(
+    [string]$RepositoryRoot,
+    [string]$ToolsLocalPath,
+    [switch]$Force = $false
+)
+
+$rootToolVersions = Join-Path $RepositoryRoot ".toolversions"
+$bootstrapComplete = Join-Path $ToolsLocalPath "bootstrap.complete"
+
+# if the force switch is specified delete the semaphore file if it exists
+if ($Force -and (Test-Path $bootstrapComplete))
+{
+    del $bootstrapComplete
+}
+
+# if the semaphore file exists and is identical to the specified version then exit
+if ((Test-Path $bootstrapComplete) -and !(Compare-Object (Get-Content $rootToolVersions) (Get-Content $bootstrapComplete)))
+{
+    exit 0
+}
+
+$initCliScript = "dotnet-install.ps1"
+$initCliLocalPath = Join-Path $ToolsLocalPath $initCliScript
+
+# blow away the tools directory so we can start from a known state
+if (Test-Path $ToolsLocalPath)
+{
+    rd -recurse $ToolsLocalPath
+}
+mkdir $ToolsLocalPath | Out-Null
+
+# download CLI boot-strapper script
+Invoke-WebRequest "https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1" -OutFile $initCliLocalPath
```

Comment:
Yes.

What I've been doing locally is hacking up init-tools.sh to just copy everything from a fixed directory (which I prepped by using dotnet-install.ps1 offline).  If there was some way to sent an environment variable that would do that, it would be great.